### ISSUE TITLE
Change the suggested MIT license layout

### ISSUE THREAD

**Issue by Retro64** [zalando/zalando-howto-open-source#18]
If the LICENSE file starts with
```
MIT License (MIT)
```
instead of 'The MIT License (MIT)', Github is able to parse the License (and putting a fancy badge on the tabs - see 'MIT' tab next to contributors on https://github.com/zalando-incubator/lib-oauth-tooling).

**Comment by lasomethingsomething** [comment]
@Retro64 Would you want to pitch this idea internally? Then others might comment.

**Comment by Retro64** [comment]
@LappleApple Do you mean via OS-Guild newsletter?

**Comment by himanshu-singh-codes** [comment]
Yes , Should start with  **MIT License (MIT)** will earn you a batch so that more people can Connect
